src/test/scala/org/broadinstitute/hail/methods/ImputeSexSuite.scala

```diff
@@ -35,16 +35,20 @@ class ImputeSexSuite extends SparkSuite {
 
   object Spec extends Properties("ImputeSex") {
 
+    val plinkSafeBiallelicVDS = VariantSampleMatrix.gen(sc, VSMSubgen.plinkSafeBiallelic)
+      .resize(1000)
+      .map(vds => vds.filterVariants { case (v, va, gs) => v.isAutosomalOrPseudoAutosomal && v.contig.toUpperCase != "X" && v.contig.toUpperCase != "Y" })
+      .filter(vds => vds.nVariants > 2 && vds.nSamples >= 2)
+
     property("hail generates same results as PLINK v1.9") =
-      forAll(VariantSampleMatrix.gen[Genotype](sc, VSMSubgen.random)) { case (vds: VariantSampleMatrix[Genotype]) =>
+      forAll(plinkSafeBiallelicVDS) { case (vds: VariantSampleMatrix[Genotype]) =>
 
         var s = State(sc, sqlContext).copy(vds = vds.copy(rdd =
           vds.rdd.map { case (v, (va, gs)) => (v.copy(contig = "X"), (va, gs)) }
             .toOrderedRDD))
 
-        s = SplitMulti.run(s)
         s = VariantQC.run(s)
-        s = FilterVariantsExpr.run(s, Array("--keep", "-c", "va.qc.AC > 0"))
+        s = FilterVariantsExpr.run(s, Array("--keep", "-c", "va.qc.AC > 1 && va.qc.AF >= 1e-8 && va.qc.nCalled * 2 - va.qc.AC > 1 && va.qc.AF <= 1 - 1e-8"))
```

Comment:
Also, the reason for excluding monomorphic variants is they don't contribute towards a test for excess heterozygosity as everyone will be homozygote.